src/pipeline.rs

```diff
@@ -382,3 +430,98 @@ pub(crate) fn prepare_view_targets(
         }
     }
 }
+
+#[derive(Resource, Clone)]
+pub struct PointCloudPlaybackControl {
+    pub playing: bool,
+    pub speed: f32,
+}
+impl Default for PointCloudPlaybackControl {
+    fn default() -> Self {
+        Self {
+            playing: true,
+            speed: 5.0,
+        }
+    }
+}
+
+impl ExtractResource for PointCloudPlaybackControl {
+    type Source = Self;
+
+    /// Defines how the resource is transferred into the "render world".
+    fn extract_resource(source: &Self::Source) -> Self {
+        source.clone()
+    }
+}
+
+pub fn prepare_animated_assets(
+    queue: Res<RenderQueue>,
+    mut assets: ResMut<RenderAssets<PointCloudAsset>>,
+    playback: ResMut<PointCloudPlaybackControl>,
```

Comment:
I think it would be more useful to have a `PointCloudPlaybackControl` for every point cloud so you can control their playback independently. I suggest making it a `Component`.

Reply:
You have a point, but doing it like this would be significantly easier. I don't think we're required to animate multiple objects at once at this time.

Reply:
OK yea if it's not a requirement, then this is good enough for now.